identity-v1/apiproxy/policies/AssignMessage.SetCorsHeaders.xml

```diff
@@ -0,0 +1,12 @@
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.SetCorsHeaders">
+  <Set>
+    <Headers>
+      <Header name="Access-Control-Allow-Origin">{request.header.origin}</Header>
+      <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, authorization</Header>
+      <Header name="Access-Control-Max-Age">3628800</Header>
```

Comment:
This is an extremely long expiry time - let's keep it fairly short. Absolute max 1 day, 5 mins-2 hours is fairly standard in most browsers.

Reply:
Per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age I'll take this down to 10 minutes (Chromium's old maximum)

Reply:
:+1: